test/rmath.jl

```diff
@@ -376,6 +398,29 @@ end
         ((Inf32,), -5f0:0.1f0:5f0),
     ])
 
+    rmathcomp_tests("signrank", [
+        ((4),-2:12),
+        ((4),-2.0:0.25:12.0),
+        ((10),-2:57),
+        #((50),-2:1277),
```

Comment:
Should this be removed? Or uncommented?

Reply:
Maybe I'll switch it to a broken test, so we can easily track if R fixes it behavior.